HQSmokeTests/testPages/data/manage_forms_page.py

```diff
@@ -44,15 +47,22 @@ def get_normal_forms(self):
         time.sleep(5)
 
     def view_normal_form(self):
-        self.wait_and_sleep_to_click(self.view_form_link)
-        self.switch_to_next_tab()
-        normal_form_data = self.driver.page_source
-        assert normal_form_data != ""  # This condition can be improvised
-        print("normal_form has data")
-        self.driver.close()
-        self.switch_back_to_prev_tab()
+        result = self.is_present_and_displayed(self.view_form_link)
+        if result == False:
+            self.select_by_value(self.select_archive_restore, "restore")
+            self.wait_to_click(self.apply_button)
+            self.restore_all_forms()
+            self.get_normal_forms()
```

Comment:
We're not asserting in this if block, for the condition when no formats are present.
@kbo001 @nsaxena03

Reply:
https://github.com/dimagi/dimagi-qa/pull/105/commits/125214317373d908f13be899960fa4d6a24499f4